lib/itk/queue/consumer.ex

```diff
@@ -330,6 +329,11 @@ defmodule ITKQueue.Consumer do
     )
 
     AMQP.Basic.reject(channel, tag, requeue: false)
+
```

Comment:
A few lines above, the log level is `Logger.info`. If we are going to change the intended error level of reject then we should be consistently about it.

Also, currently itk-email will reject messages for **normal** processing errors (like duplicate scheduled message uuid), it will have to stop using reject and simply ack the message.

Reply:
@maruika-wei I didn't actually get this part: 
```
Also, currently itk-email will reject messages for normal processing errors (like duplicate scheduled message uuid), it will have to stop using reject and simply ack the message.
```

Do you mean it's normal to fail consuming messages in itk-email and that it'd be too noisy to be reported?